Makefile

```diff
@@ -71,6 +71,11 @@ E2E_K8S_VERSIONS ?= 1.26.12 1.27.9 1.28.5 1.29.0
 # Default will delete default kind cluster
 KIND_CLUSTER_NAME ?= kind
 
+# Number of processes to use during integration tests to run specs within a
+# suite in parallel. Suites still run sequentially. User may set this value to 1
```

Comment:
"Suites still run sequentially" seems orthogonal to the param. Is there another ginkgo option that let's us control parallel execution of suites?

Reply:
I don't think so. The [Ginkgo documentation](https://onsi.github.io/ginkgo/#running-multiple-suites) states "when there are multiple suites to run Ginkgo...  always runs them sequentially"

Reply:
Thanks for the ref. In that case I'm ok with  "Suites still run sequentially.".